test-registry.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020 The Cockroach Authors
```

Comment:
Do we need this file?

Reply:
Yes... this should be kept, is the CR that register the operator on OperatorHub, but I will add an issue with the automation of the docker bundle and, package organization